python/kserve/Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 
 dev_install:
-	poetry install --with test,dev --extras "storage"
```

Comment:
This is needed to fix build

Reply:
This is also addressed in [this PR](https://github.com/kserve/kserve/pull/3271). How about merging that PR first ?

Reply:
I had to fix it in my PR to unblock the release. Otherwise, I cannot pass CI build.

Reply:
Happy to revert my changes and rebase if it's merged first.